codebundles/k8s-tail-logs-dynamic/sli.robot

```diff
@@ -0,0 +1,131 @@
+*** Settings ***
+Documentation       Measures the number of exception stacktraces present in an application's logs over a time period.
+Metadata            Author    jon-funk
+Metadata            Display Name    Kubernetes Tail Application Logs
+Metadata            Supports    Kubernetes,AKS,EKS,GKE,OpenShift,GoLang,Json,Python,CSharp,Django,Node,Java,FastAPI
+
+Library             BuiltIn
+Library             RW.Core
+Library             RW.CLI
+Library             RW.K8sApplications
+Library             RW.platform
+Library             OperatingSystem
+
+Suite Setup         Suite Initialization
+
+
+*** Tasks ***
+Tail `${CONTAINER_NAME}` Application Logs For Stacktraces
+    [Documentation]    Tails logs and organizes output for measuring counts.
+    [Tags]    resource    application    workload    logs    state    exceptions    errors
+    ${cmd}=    Set Variable
+    ...    ${KUBERNETES_DISTRIBUTION_BINARY} --context=${CONTEXT} -n ${NAMESPACE} logs -l ${LABELS} --tail=${MAX_LOG_LINES} --max-log-requests=10 --limit-bytes=${MAX_LOG_BYTES} --since=${LOGS_SINCE} --container=${CONTAINER_NAME}
+    IF    $EXCLUDE_PATTERN != ""
+        ${cmd}=    Set Variable
+        ...    ${cmd} | grep -Ev "${EXCLUDE_PATTERN}" || true
+    END
+
+    ${logs}=    RW.CLI.Run Cli
+    ...    cmd=${cmd}
+    ...    show_in_rwl_cheatsheet=true
+    ...    render_in_commandlist=true
```

Comment:
we don't typically add SLI code into cheatsheet data (though I think the cheatsheet will omit it)